tracer/src/Datadog.Tracer.Native/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
```

Comment:
Is this of any use?

Reply:
thanks, removed in  https://github.com/DataDog/dd-trace-dotnet/pull/4523/commits/1c21c9e7834e9240810546e836aef6c1ecdc2350